### PR TITLE
MO DeformableConv: fix a set of attributes

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
 - *fix a set of attributes for opset < opset8*

ticket: 56576, 56582

Code:

- [x]  Comments
- [x]  Code style (PEP8)
- [x]  Transformation generates reshape-able IR
- [x]  Transformation preserves original framework node names

Validation:

- [ ]  Unit tests
- [x]  Framework operation tests - N/A
- [x]  Transformation tests N/A
- [x]  Model Optimizer IR Reader check N/A

Documentation:

- [x]  Supported frameworks operations list - N/A
- [x]  Guide on how to convert the public model - N/A
- [x]  User guide update - N/A

### CODE REVIEW

File: model-optimizer/mo/ops/deformable_convolution.py [hunk 1]
```diff
@@ -27,4 +27,9 @@ def __init__(self, graph: Graph, attrs: dict):
     def backend_attrs(self):
         # the same attributes as in a regular convolution and additional attributes 'deformable_group', 'group'
         # and 'bilinear_interpolation_pad'
-        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group', 'bilinear_interpolation_pad']
+        version = self.get_opset()
+        if version == 'opset8':
+            return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group',
+                                                                  'bilinear_interpolation_pad']
+        else:
+            return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group']
```
Comment (lazarevevgeny) [inline model-optimizer/mo/ops/deformable_convolution.py:35]: May be a bit cleaner code:

```python3
attrs = ['deformable_group', 'group']
if self.get_opset() == 'opset8':
    attrs.append('bilinear_interpolation_pad')
return attrs
```

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

**Comment by rkazants** [comment]
@itikhono, could you add follow MO Code Review template and fill?

File: model-optimizer/mo/ops/deformable_convolution.py (hunk 1 above)
Comment (itikhono) [reply model-optimizer/mo/ops/deformable_convolution.py]: do we have any relevant test suite to add a test?

Comment (rkazants) [reply model-optimizer/mo/ops/deformable_convolution.py]: Unfortunately, no

**Review by rkazants — APPROVED** [review]

**Comment by pavel-esir** [comment]
I had a network with DeformableConv and when `DeformableConv-8` was turned on I had performance degradation since CPU was using `evaluate` for inference. Looks like because of some attribute `v8` was not successfully downgraded in IE. And even cherry-picking the current PR does not solve the problem. Moreover error occurs on nG side during validation.

@itikhono could you please check your changes on the model in ticket 59003 ?

**Comment by itikhono** [comment]
> I had a network with DeformableConv and when `DeformableConv-8` was turned on I had performance degradation since CPU was using `evaluate` for inference. Looks like because of some attribute `v8` was not successfully downgraded in IE. And even cherry-picking the current PR does not solve the problem. Moreover error occurs on nG side during validation.
> 
> @itikhono could you please check your changes on the model in ticket 59003 ?

ok, I will check.

**Review by lazarevevgeny — APPROVED** [review]

File: model-optimizer/mo/ops/deformable_convolution.py [hunk 2]
```diff
@@ -27,4 +27,7 @@ def __init__(self, graph: Graph, attrs: dict):
     def backend_attrs(self):
         # the same attributes as in a regular convolution and additional attributes 'deformable_group', 'group'
         # and 'bilinear_interpolation_pad'
-        return Convolution(self.graph, {}).backend_attrs() + ['deformable_group', 'group', 'bilinear_interpolation_pad']
+        attrs = ['deformable_group', 'group']
+        if self.get_opset() == 'opset8':
+            attrs.append('bilinear_interpolation_pad')
+        return attrs
```
Comment (lazarevevgeny) [inline model-optimizer/mo/ops/deformable_convolution.py:33]: How about `Convolution(self.graph, {}).backend_attrs()` ? We you have removed them?

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/mo/ops/deformable_convolution.py (hunk 2 above)
Comment (itikhono) [reply model-optimizer/mo/ops/deformable_convolution.py:33]: oops, my apologies

Comment (itikhono) [reply model-optimizer/mo/ops/deformable_convolution.py:33]: fixed

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**